esp-wifi/src/ble/btdm.rs

```diff
@@ -257,7 +248,7 @@ unsafe extern "C" fn queue_recv(queue: *const (), item: *const (), block_time_ms
             }
 
             if block_time_ms != OSI_FUNCS_TIME_BLOCKING
-                && crate::timer::get_systimer_count() > end_time
+                && crate::timer::get_systimer_count() > end_time_ticks
```

Comment:
This won't handle the wrap-around of the systimer count

Reply:
can we let this slide until https://github.com/esp-rs/esp-wifi/pull/316 happens?